sys/include/net/gnrc.h

```diff
@@ -155,26 +155,46 @@
  * Transmitting Packets
  * --------------------
  *
- * To relay the @ref net_gnrc_pkt to interested threads the @ref
- * net_gnrc_netapi offers a dispatch functionality. The following example
+ * A packet is transmitted by relaying it to threads interested in handling (and
+ * dispatching) packets of its type. To do this, the @ref
+ * net_gnrc_netapi offers a dispatch functionality called @ref
+ * gnrc_netapi_dispatch_send().
+ *
+ * The following example
```

Comment:
`Jgqq` (vim2human: please join and with following line and truncate to 72 chars again please)